bindings/java/Cargo.toml

```diff
@@ -129,6 +130,8 @@ services-wasabi = ["opendal/services-wasabi"]
 services-mysql = ["opendal/services-mysql"]
 services-mongodb = ["opendal/services-mongodb"]
 services-sqlite = ["opendal/services-sqlite"]
+services-libsql = ["opendal/services-libsql"]
+test-retry = []
```

Comment:
It doesn't make sense to add this feature. We need a better solution from java binding side.

Reply:
cc @tisonkun @G-XD

Reply:
I think we can keep the same behavior with the rust core behavior test. We retry it during the core side rather than the test side

Reply:
https://github.com/apache/incubator-opendal/blob/main/core/tests/behavior/utils.rs#L91

Reply:
> I think we need to keep the same behavior with the rust core behavior test. We retry it during the core side rather than the test side

This conclusion is correct, but not in this way.

Reply:
> This conclusion is correct, but not in this way.

In my thought, we have already made a special operator with some layers in core test https://github.com/apache/incubator-opendal/blob/main/core/tests/behavior/utils.rs#L80-L99.

We need to make sure we use the operator with the same behavior in Java test(maybe python test in the future)

So why not we init the different operator depend on the feature flag?

Reply:
In other way, maybe we can export the layer API to Java binding?

Reply:
Why do we need retry here?

Reply:
Just for this case:

Libsql may crash sometimes during the test. The code is correct, but the server crashed, so we need to ignore some unnecessary test failures.

BTW, the rust test code also failed because of the libsql server crash if we remove the retry layer.

Reply:
I have a rough idea to support java binding use Retry layer. Let me prototype it in this week.

Reply:
I think not just RetryLayer. Maybe we need to export layers we can to use in Rust core to other language bindings